tests/FSharp.Compiler.ComponentTests/Signatures/TestHelpers.fs

```diff
@@ -1,20 +1,15 @@
 ﻿module Signatures.TestHelpers
 
 open System
-open FsUnit
+open Xunit
 open FSharp.Test.Compiler
 
 let prependNewline v = String.Concat("\n", v)
 
-let equal x =
-    let x =
-        match box x with
-        | :? String as s -> s.Replace("\r\n", "\n") |> box
-        | x -> x
-
-    equal x
+let assertEqualIgnoreLineEnding (x: string) (y: string) =
+    Assert.Equal(x, y, ignoreLineEndingDifferences = true)
```

Comment:
This is doing the same as before, just using the framework means and having an honest function name.